legal-api/requirements.txt

```diff
@@ -60,4 +60,8 @@ reportlab==3.6.12
 html-sanitizer==2.4.1
 lxml==5.2.2
 git+https://github.com/bcgov/business-schemas.git@2.18.31#egg=registry_schemas
-git+https://github.com/bcgov/lear.git#egg=sql-versioning&subdirectory=python/common/sql-versioning
+
+# install updated sql-versioning from feature branch for development, 
+# TODO - change it back when the db versioning feature branch merged to main
+git+https://github.com/bcgov/lear.git@feature-db-versioning#egg=sql-versioning&subdirectory=python/common/sql-versioning  
+# git+https://github.com/bcgov/lear.git#egg=sql-versioning&subdirectory=python/common/sql-versioning
```

Comment:
Installing the updated custom sql-versioning package from the `feature-db-versioning` branch rather then from `main`, so people can use the latest code for development.

@leodube-aot @argush3 Let me know you don't think it's a good idea. (same to the requirements.txt for emailer)

Reply:
Kevin is going to update the `main` branch sql-versioning package so we can install it from there